recipe/meta.yaml

```diff
@@ -37,16 +31,23 @@ requirements:
 test:
   imports:
     - hyperas
+  commands:
+    - pip check
```

Comment:
ow, that is interesting, ``pip check`` is not working well for this package...
at least it is not detecting ``jupyter``